### PR TITLE
Feat: Tags

### PR DESCRIPTION
**Pull request by WesleyFelipeG**
# Change Type

Mark with an "x" in the fields you want to select:

- [ ] Bug Fix (non-breaking change that fixes an issue)
- [ ] New Feature (non-breaking change that adds functionality)
- [ ] Chore (documentation, packages, or tests updates, nothing that affects the end user directly)
- [ ] Release (new version of the application - for production only)

## Description

...

## Screenshots

...

## Tasks

- [task-id](task-link) or N/A

## Checklist

Mark with an "x" in the fields you want to select:

- [ ] My changes are 400 lines or less
- [ ] I have performed a self-review of my own code
- [ ] The existing tests and linter pass locally with my changes
- [ ] I have commented my code in hard-to-understand areas (if applicable)
- [ ] I have created tests for my fix or feature (if applicable)

## Dependencies

This pull request has dependencies on the following others:

- link-to-dependency PR or N/A

### CODE REVIEW

File: src/services/post/CreateTagService.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+     /**
+ * Creates a new tag in the database.
+ *
+ * @param {CreateTagData} data - The data for the tag being created.
+ * @param {string} tagId - The Id for the tag being created.
+ * @returns {Promise<ReturnTagData>} - A promise that resolves to the created post data, including tags
+ * 
+ * @throws {Error} - Throws an error if the post creation fails.
+ */
+/*    async create(data: CreateTagData, userId: string): Promise<ReturnPostData> {
```
Comment (gazinmarks) [inline src/services/post/CreateTagService.ts:15]: Função da classe está comentada. Não está sendo usada?

File: src/services/post/CreateTagService.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import prismaClient from '../../prisma';
+//import { CreateTagData, ReturnTagData } from "../../interfaces/post/tagType";  <--- Coisar depois
```
Comment (gazinmarks) [inline src/services/post/CreateTagService.ts:2]: Sugestão: Retirar se não estiver sendo usado

Comment (WesleyFelipeG) [reply src/services/post/CreateTagService.ts:2]: falta criar o tagType, vou usar essa rota quando for fazer ele

File: src/services/post/CreateTagService.ts (hunk 1 above)
Comment (WesleyFelipeG) [reply src/services/post/CreateTagService.ts:15]: A parte comentada é revisar a construção disso com alguém do back depois

**Pull request merged**